### PR TITLE
implementation of the zipkin converter started

### PR DESCRIPTION
**Pull request by JohannesKonings**
related to #5

### CODE REVIEW

File: src/zatt_converter/zcl_att_conv_zipkin.clas.abap [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    DATA(zipkin_json) = |[\{| &&
+                           |"id": "dummy",| &&
+                           |"traceId": "{ trace_id }",| &&
+                           |\}]|.
+
+    converted_trace-trace_id = trace_id.
+    converted_trace-json = zipkin_json.
+
+*  [{
+* "id": "11123456",
```
Comment (Goala) [inline src/zatt_converter/zcl_att_conv_zipkin.clas.abap:51]: What's with all that json? 🤔

Comment (JohannesKonings) [reply src/zatt_converter/zcl_att_conv_zipkin.clas.abap:51]: That's the reminder that it's not finished. Not good, but maybe better to have some progress.

**Comment by JohannesKonings** [comment]
added some further progress. still not finished, but every step count 😄

**Review by Goala — APPROVED** [review]
lgtm

**Pull request merged**